packages/webpack-cli/lib/utils/Compiler.js

```diff
@@ -111,13 +111,12 @@ class Compiler {
             // https://github.com/webpack/webpack/blob/master/lib/index.js#L267
             // https://github.com/webpack/webpack/blob/v4.44.2/lib/webpack.js#L90
             const ValidationError = webpack.ValidationError ? webpack.ValidationError : webpack.WebpackOptionsValidationError;
-            process.stdout.write('\n');
             // In case of schema errors print and exit process
             // For webpack@4 and webpack@5
             if (err instanceof ValidationError) {
-                logger.error(err.message);
+                logger.error(`\n ${err.message}`);
```

Comment:
Extra space between `\n ` and messages, it is expected? I think no

Reply:
Not expected but some extra space is added in windows CI for some reason 😫